configuration/pih/scripts/global/liberia.js

```diff
@@ -0,0 +1 @@
+../liberia.js
```

Comment:
Instead of moving liberia.js and liberia.css to the new "global" directories, I just created a symlinks to the exiseting liberia.js and liberia.css in new "global" subdirectories.  The reasoning for this is that until we update the PIH EMR release version in Liberia, the liberia.js and liberia.css files *should* still live in the top-level "scripts" and "styles", so until that happens we need to maintain the files in both locations.  Once Liberia is updated, we can move the files to the "global" directories and remove the symlink.